api/v1/deletion_policy.go

```diff
@@ -0,0 +1,14 @@
+package v1
+
+// DeletionPolicy specifies the way LINSTOR Satellite deletions are handled.
```

Comment:
"LINSTOR Satellite deletions" seems a bit ambiguous to me. Does it mean when the node is removed from Kubernetes? When the node is deleted from LINSTOR? When the LinstorSatellite custom resource is deleted? When the LinstorSatelliteConfiguration is changed such that a node is no longer included? When the Pod or DaemonSet is removed behind our back?

Reply:
I think I've cleared it up with a more verbose explaination :)